platforms/org.eclipse.mita.platform.xdk110/src/org/eclipse/mita/platform/xdk110/io/SDCardGenerator.xtend

```diff
@@ -161,18 +157,17 @@ class SDCardGenerator extends AbstractSystemResourceGenerator {
 			return retcode;
 		''')

```

Comment:
One last thing, then you can merge: please add an include to "MitaExceptions.h".